corehq/apps/userreports/expressions/specs.py

```diff
@@ -291,6 +291,70 @@ def __str__(self):
                                   str(self._value_expression))
 
 
+class IndexedCaseExpressionSpec(JsonObject):
+    type = TypeProperty('indexed_case')
+    case_expression = DictProperty(required=True)
+    index = StringProperty(required=False)
+
+    def configure(self, case_expression, context):
+        from corehq.apps.userreports.expressions import ExpressionFactory
+        self._case_expression = case_expression
+
+        index = self.index or 'parent'
+        spec = {
+            'type': 'related_doc',
```

Comment:
Since this is doesn't have any extra python code and it's just a shorthand expression for other UCR expression can it be added to https://github.com/dimagi/commcare-hq/blob/master/custom/ucr_ext/expressions.py instead?